core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -35,6 +36,25 @@
  */
 @Recorder
 public class LoggingSetupRecorder {
+
+    static final boolean IS_WINDOWS = System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("win");
+
+    /**
+     * <a href="https://conemu.github.io">ConEmu</a> ANSI X3.64 support enabled,
+     * used by <a href="https://cmder.net/">cmder</a>
+     */
+    static final boolean IS_CON_EMU_ANSI = "ON".equals(System.getenv("ConEmuANSI"));
+
+    static final boolean IS_CYGWIN = IS_WINDOWS
+            && System.getenv("PWD") != null
+            && System.getenv("PWD").startsWith("/")
+            && !"cygwin".equals(System.getenv("TERM"));
```

Comment:
if this is correct, maybe it requires a comment. It feels weird to check the `TERM` is not `cygwin` for Cygwin.

Reply:
this is taken literally as is from jansi assuming they were right :)

Reply:
for what it is worth - TERM=xterm on default cygwin.

Reply:
https://github.com/fusesource/jansi/commit/bb3d538315c44f799d34fd3426f6c91c8e8dfc55 is when it was added. 

Googling around a bit seems `TERM=cygwin` used mainly to trick some other tools to think you are running cygwin when you aren't :)

Reply:
added comment and consider it correct as from jansi used by mvn and others.